File: cfgs/java_cfgs_foo/MINIMIZE_THE_SUM_OF_DIGITS_OF_A_AND_B_SUCH_THAT_A_B_N.java
Nodes
1;special;;;
2;statement;int sum = 0;14;
3;conditional;foo_1 > 0;15;
4;statement;sum += foo_1 % 10;16;
5;statement;n /= 10;17;
6;conditional;foo_1 == 1;19;
7;statement;return 10;19;
8;exit;;;
9;statement;return foo_1;20;
Edges
0;2;
1;3;
2;6;4;
3;5;
4;3;
5;7;9;
6;8;
7;
8;8;

File: cfgs/pythoncfgs/MINIMIZE_THE_SUM_OF_DIGITS_OF_A_AND_B_SUCH_THAT_A_B_N.py
Nodes
1;statement;foo_1 = 0;8;
2;conditional;foo_1 > 0;9;
3;conditional;foo_1 == 1;12;
4;statement;return foo_1;14;
5;statement;return 10;13;
6;statement;foo_1 += foo_2 % 10;10;
7;statement;foo_1 //= 10;11;
8;exit;;
Edges
0;2;
1;3;6;
2;4;5;
3;8;
4;8;
5;7;
6;2;
7;